xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -4833,7 +4833,7 @@ core.viewers.diff.metadata.space=Space
 core.create.spaceTitle=Create Space
 core.create.space=Space Name
 core.create.space.hint=Name of the new space
-core.create.space.template.hint=Template to use for the homepage of the new space
+core.create.space.template.hint= Template to use for the homepage of the new space
```

Comment:
This looks wrong to have a leading white space. Could you elaborate why it's needed? If there's really a need then I' d say the issue is not with the translation but with the place where the translation is used.

Reply:
Already fixed in https://github.com/xwiki/xwiki-platform/commit/3d30b22981eb4e4db8c6472f9924d6ec054c5f70